src/libPMacc/CMakeLists.txt

```diff
@@ -23,11 +23,26 @@
 ################################################################################
 # PMacc tests
 ################################################################################
-cmake_minimum_required(VERSION 3.3)
-project ("PMacc-tests")
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Changing requirements hidden inside a pull request which solves an other issue should avoided.